lib/tasks/workers.rake

```diff
@@ -28,4 +28,15 @@ namespace :workers do
       Sneakers::Runner.new(workers).run
     end
   end
+
+  namespace :index_documents do
+    desc 'run an ElasticsearchIndexJob'
+    task run: :environment do
+      silence_warnings do
+        Rails.application.eager_load! unless Rails.application.config.eager_load
```

Comment:
We may not need to call eager_load! here, since we're not using inflection to find the job classes. Removal can be explored in a future PR.